apps/web/modules/organization/settings/teams/lib/invite.ts

```diff
@@ -216,7 +112,7 @@ export const inviteUser = async ({
   organizationId: string;
   invitee: TInvitee;
   currentUserId: string;
-}): Promise<TInvite> => {
+}): Promise<Pick<Invite, "id">> => {
```

Comment:
can we also just return the `id` from here as a string like we did in the other PR?